Code/Tools/LuaIDE/Source/LUA/LUADebuggerComponent.h

```diff
@@ -90,12 +94,10 @@ namespace LUADebugger
         virtual void DesiredTargetChanged(AZ::u32 newTargetID, AZ::u32 oldTargetID);
         //////////////////////////////////////////////////////////////////////////
 
-        //////////////////////////////////////////////////////////////////////////
-        // TmMsgBus
-        virtual void OnReceivedMsg(AzFramework::TmMsgPtr msg);
-        //////////////////////////////////////////////////////////////////////////
-
         static void Reflect(AZ::ReflectContext* reflection);
+
+     private:
+        AzFramework::IRemoteTools* m_remoteTools;
```

Comment:
Can you initialize this to `nullptr`?
This doesn't actually have a determinstic value unless `Activate` call.
If it is never called then m_remoterTools is indeterminate and can't be checked against nullptr.

```suggestion
        AzFramework::IRemoteTools* m_remoteTools = nullptr;
```